lib/fluent/plugin/bigquery/schema.rb

```diff
@@ -66,12 +66,32 @@ def format_one(value)
       end
     end
 
+    class JsonFieldSchema < FieldSchema
+      def type
+        :json
+      end
+
+      def format_one(value, is_load: false)
+        if is_load
+          value
```

Comment:
It's very annoying.
But otherwise, when the load API is used, it will be imported with escapes.